mappings/net/minecraft/client/network/PlayerListEntry.mapping

```diff
@@ -26,6 +34,10 @@ CLASS net/minecraft/class_640 net/minecraft/client/network/PlayerListEntry
 	METHOD method_2970 setLatency (I)V
 		ARG 1 latency
 	METHOD method_2971 getDisplayName ()Lnet/minecraft/class_2561;
+	METHOD method_2972 setScore (I)V
```

Comment:
Since this method is called only when the render type of the objective is `RenderType.HEARTS`, I'd name it `setHealth`. Also I wonder how this method is different from `method_2965`.

Reply:
Yeah, when I saw the code it was like "hm what" - setHealth seems fine.